fish_prompt.fish

```diff
@@ -1,8 +1,5 @@
 function fish_prompt
   test $status -ne 0;
-    and set -l colors 600 900 c00
-    or set -l colors 333 666 aaa
-
```

Comment:
Did you accidentally delete these lines? They set the colour for the ">>>" chars in the prompt depending on the last command's exit status code.